MapboxNavigation/RouteVoiceController.swift

```diff
@@ -28,10 +28,15 @@ public class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
      */
     public var volume: Float {
         get {
+            guard let audioPlayer = audioPlayer else {
+                return 1
+            }
             return audioPlayer.volume
         }
         set {
-            audioPlayer.volume = newValue
+            if let audioPlayer = audioPlayer {
+                audioPlayer.volume = newValue
+            }
```

Comment:
@frederoni have any ideas how to handle volume here? If `AVAudioPlayer ` is not inited yet, you cannot change the volume.